lib/ovh.js

```diff
@@ -654,6 +663,17 @@ var https = require('https'),
     return '$1$' + crypto.createHash('sha1').update(s.join('+')).digest('hex');
   };
 
+  /**
+   * Change the custom HTTPS agent used for reaching the OVH servers
+   *
+   * @param {Object} httpsAgent: the custom HTTPS agent
+   */
+  Ovh.prototype.setCustomHTTPSAgent = function (httpsAgent) {
+    if (typeof(httpsAgent) === 'undefined' || typeof(httpsAgent) === 'object') {
+      this.httpsAgent = httpsAgent;
+    }
```

Comment:
How do you unset an existing agent ? This said, I can't find a use-case where a setter is needed at all. When would one want to switch proxies at runtime ?